### DISCUSSION TITLE
Error when reading 'lib/generated_bridge.dart': No such file or directory

### DISCUSSION THREAD

**Discussion by doujd** [rustdesk/rustdesk#4511] [Q&A]
Hello, I'm sorry to bother you.

After importing the fluuter project on Mac, I found the file ‘/rustdesk/flutter/lib/models/native_model.dart’. There is 'import '../generated_bridge.dart';'. But in fact, I found all of them in the GIT project and didn't have this file 'generated_bridge.dart'

**Comment by wangfeng211** [comment]
I also encountered this problem。

**Comment by beelchester** [comment]
@doujd @wangfeng211 
Did you try generating flutter rust bridge files beforehand? If not, you need to do that.
You can simply run this script and it will do the job for you.
[rustdesk/flutter/run.sh](https://github.com/rustdesk/rustdesk/blob/master/flutter/run.sh)

**Reply by TBG-FR** [reply]
What args should I use with the `run.sh` script ?

**Reply by larssn** [reply]
Args are optional, and are postfixed to the `flutter run` command.
